velox/common/tests/BitUtilTest.cpp

```diff
@@ -416,6 +416,43 @@ TEST_F(BitUtilTest, forEachBit) {
   expectedIndex = 3;
   forEachUnsetBit(data, 2, totalBits - 3, calledOnEveryOther);
   EXPECT_EQ(totalBits - 3, expectedIndex);
+
+  auto numEntries = 100;
+  uint32_t size = bits::nwords(numEntries);
+  std::vector<uint64_t> vec(size);
+  uint64_t* vecPtr = vec.data();
+
+  // Test out iterating over all entries when true
+  bits::fillBits(vecPtr, 0, numEntries, true);
+  int count = 0;
+  bits::forEachBit(vecPtr, 0, numEntries, true, [&](auto row) {
```

Comment:
nit: consider introducing temp variable for the lambda to avoid copy-paste

```
auto countEach = [&](auto row) {
    ASSERT_EQ(row, count);
    count++;
  };

bits::forEachSetBit(rawData, 0, totalBits, countEach);
ASSERT_EQ(totalBits, count);

...
bits::forEachUnsetBit(rawData, 0, totalBits, countEach);
ASSERT_EQ(totalBits, count);
```